src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_snapshot_commands.py

```diff
@@ -179,9 +177,9 @@ def test_get_snapshot(self):
         snapshot_from_id = self.cmd("az netappfiles snapshot show --ids %s" % snapshot['id']).get_output_in_json()
         assert snapshot_from_id['name'] == account_name + '/' + pool_name + '/' + volume_name + '/' + snapshot_name
 
-    @unittest.skip('This test requires southcentralusstage and cannot be run with public cloud.')
+    @serial_test()
```

Comment:
As long as this test doesn't manipulate same files with other tests, `serial_test` is not necessary.

Reply:
serial_test annotation has been removed